api/handlers_security.go

```diff
@@ -0,0 +1,91 @@
+package api
```

Comment:
not a big deal, but let's name the file `handlers_security_groups.go` or `handlers_sgs.go`